packages/react-instantsearch-dom-geo/src/GeoSearch.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+// @TODO: Update this import when the package is correctly split:
+// import { createClassNames } from 'react-instantsearch-dom';
+import createClassNames from '../../react-instantsearch/src/components/createClassNames';
+import { LatLngPropType } from './propTypes';
+import Provider from './Provider';
+import GoogleMaps from './GoogleMaps';
+
+const cx = createClassNames('GeoSearch');
+
+class GeoSearch extends Component {
+  static propTypes = {
+    google: PropTypes.object.isRequired,
```

Comment:
At some point if we need an object that is not nested under the `maps` namespace it will be a breaking change. Not sure that will happen one day but anyway the overhead is inexistent.